scripts/main.coffee

```diff
@@ -9,15 +9,25 @@
 #   These are from the scripting documentation: https://github.com/github/hubot/blob/master/docs/scripting.md
 
 module.exports = (robot) ->
-    
+
+
+  config = require('../config.json')
+
   messages = require('./messages.js').messages
+
   messages.setAuth((err, res) ->
       console.log("Logged in: " + res[1].title)
   )
 
-  robot.hear conf.keys, (res) ->
-      messages.sendMessage(res.message, (r) ->
-          console.log("Added at: " + r.updated))
+  console.log(config.docs[config.monitor['+todo'].dest])
+
+  robot.hear /.*/i, (res) ->
+      message = res.message.text.split(' ', 1)
+      for key, val of config.monitor
+          if(key == message[0])
```

Comment:
hmmm, but i think more preferrable format would be to type command in the beginning, but if you want also to fetch it in the end -- i can do this